### PR TITLE
Create aaron.json

### PR DESCRIPTION
**Pull request by Adriichu**

### CODE REVIEW

**Comment by wclarkey** [comment]
```json
    "target": {
      "record-type": {
         "name": "aaron", 
         "value": "aaron-web.pages.dev"
       }
    },
```
    
Please change `"record-type"` to the relevent record type. (https://is-a-good-dev.github.io/docs/format.html)

File: sub-logs/aaron.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+    "repo": "https://github.com/Adriichu/is-a-good-dev",
+  
+    "owner": {
+      "username": "Adriichuuu",
+      "email": "aaron@proton.me"
+    },
+  
+    "target": {
+      "record-type": {
```
Comment (WilliamDavidHarrison) [inline sub-logs/aaron.json:10]: Please change `record-type` to `CNAME`, as this looks like the type of record you are using.

**Review by WilliamDavidHarrison — CHANGES_REQUESTED** [review]

File: sub-logs/aaron.json (hunk 1 above)
Comment (Adriichu) [reply sub-logs/aaron.json:10]: Sorry, I completely forgot about that.

**Review by WilliamDavidHarrison — APPROVED** [review]

**Review by KaminoCodes — APPROVED** [review]

**Comment by WilliamDavidHarrison** [comment]
I'm the first helper to merge a PR!

**Comment by KaminoCodes** [comment]
ok  now point out who cares

**Pull request merged**